ShopGo/LoginScreen/LoginSignupExtenstion.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  LoginSignupExtenstion.swift
+//  ShopGo
+//
+//  Created by Quratulain on 7/26/23.
+//
+
+import UIKit
+
+extension LoginSignupVC {
+    // function for radius of login button inside the view
+    func loginButtonRadius() {
```

Comment:
In the future we can do much more setup in the function besides radius so rename this to setupLoginButton().

Reply:
already created a custom class for it, this function is only going to be used for the login button.